maker-bundle/src/Resources/skeleton/content-element/ContentElement.tpl.php

```diff
@@ -0,0 +1,31 @@
+<?= "<?php\n" ?>
+
+declare(strict_types=1);
+
+namespace App\Controller\ContentElement;
+
+use Contao\ContentModel;
+use Contao\CoreBundle\Controller\ContentElement\AbstractContentElementController;
+<?php if (PHP_VERSION_ID >= 80000): ?>
```

Comment:
Checking the current PHP version can cause unwanted effects if the project itself allows a lower verion. Can we instead read the php requirement from the root *composer.json* of the project?

Reply:
There is a helper class in the maker bundle that reads the `composer.lock` file.

https://github.com/symfony/maker-bundle/blob/716eee9c8b10b33e682df1b7d80b9061887e9691/src/Util/PhpCompatUtil.php#L32-L37

However, the only check `platform-overrides`. Would this suffice in our case, too?

Reply:
I think it would be sufficient for this edge case, because if someone has this problem in a project, they can then fix it by adding `platform-overrides` to their composer.json.

Reply:
Implemented in 39c3af3735efacaedeca0f35df7c599345d23a04.